docs/content/flink/procedures.md

```diff
@@ -489,17 +489,21 @@ All available procedures are listed below.
          -- for Flink 1.18<br/>
          -- clear table with purge files directly.<br/>
          CALL [catalog.]sys.purge_files('identifier')<br/><br/>
+         -- only check what dirs will be deleted, but not really delete them.
+         CALL [catalog.]sys.purge_files('identifier', true)<br/><br/>
          -- for Flink 1.19 and later<br/>
          -- clear table with purge files directly.<br/>
-         CALL [catalog.]sys.purge_files(`table` => 'default.T')<br/><br/>
+         CALL [catalog.]sys.purge_files(`table` => 'default.T', `dry_run` => true)<br/><br/>
```

Comment:
There seems to be a similar document for Spark, and this parameter could be recorded in that document as well.